packages/squidex/schema/schemas/research-outputs.json

```diff
@@ -10,7 +10,9 @@
       "label": "Research Outputs",
       "validateOnPublish": false
     },
-    "scripts": {},
+    "scripts": {
+      "update": "const modified = Object.keys(ctx.data)\n        .filter(key => JSON.stringify(ctx.data[key]) !== JSON.stringify(ctx.oldData[key]))\n        .filter(key => key !== 'publishDate' && key !== 'adminNotes');\n        \nif (modified.length > 0) {\n    ctx.data.test2.iv = JSON.stringify(modified);\n    ctx.data.lastUpdatedPartial.iv = new Date().toISOString().split('.')[0]+\"Z\";\n    replace();\n}\n\n"
```

Comment:
```typescript
const modified = Object.keys(ctx.data)
  .filter(
    (key) => JSON.stringify(ctx.data[key]) !== JSON.stringify(ctx.oldData[key])
  )
  .filter((key) => key !== "publishDate" && key !== "adminNotes");

if (modified.length > 0) {
  ctx.data.test2.iv = JSON.stringify(modified);
  ctx.data.lastUpdatedPartial.iv = new Date().toISOString().split(".")[0] + "Z";
  replace();
}
```

Reply:
Can we remove the test bit?
Also, I don't think you need to remove the miliseconds from the date. https://support.squidex.io/t/implemented-support-iso-8601-missing-miliseconds/2191